package.json

```
@@ -45,15 +45,18 @@
   "scripts": {
     "docs": "typedoc .",
     "prepare": "husky install",
-    "test": "lerna run test",
+    "test": "lerna run test --ignore \"**/*/*indexer*\"",
```

```suggestion
    "test": "lerna run test --parallel --ignore \"**/*/*indexer*\"",
```

maybe we can run tests in parallel mode since the tests are independent?